src/backend/joanie/payment/models.py

```diff
@@ -344,12 +344,31 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class CreditCardManager(models.Manager):
+    """Custom manager for `CreditCard` model"""
+
+    def get_card_for_owner(self, token, user_id):
```

Comment:
Related to my previous comment, you should use the `id` and not the `token` here

Reply:
Indeed, thanks :pray: